api/marketdata.go

```diff
@@ -1,28 +1,28 @@
 package api
 
 import (
+	"encoding/json"
 	"fmt"
 	"github.com/gin-gonic/gin"
 	"github.com/spf13/viper"
 	"github.com/trustwallet/blockatlas/coin"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 	"github.com/trustwallet/blockatlas/pkg/ginutils"
-	"github.com/trustwallet/blockatlas/pkg/ginutils/gincache"
 	"github.com/trustwallet/blockatlas/pkg/logger"
 	"github.com/trustwallet/watchmarket/market"
 	"github.com/trustwallet/watchmarket/pkg/watchmarket"
 	"github.com/trustwallet/watchmarket/services/assets"
+	"github.com/trustwallet/watchmarket/services/caching"
 	"github.com/trustwallet/watchmarket/storage"
 	"math/big"
 	"net/http"
 	"strconv"
 	"strings"
-	"time"
 )
 
 const (
 	defaultMaxChartItems = 64
-	tokenTWT             = "TWT-8C2"
+	minute               = 60
```

Comment:
non-blocking: I think this is redundant - put the actual value here e.g. `cacheMaxAgeMinutes` or sth but not the definition of minute in seconds